lib/foreman/telemetry_sinks/prometheus_sink.rb

```diff
@@ -7,6 +7,8 @@ def initialize(opts = {})
       require 'prometheus/client/data_stores/direct_file_store'
       # Set multiprocess-friendly data store
       FileUtils.mkdir_p(PROMETHEUS_STORE_DIR)
+      # but clean it during startup as files will accomodate over time
```

Comment:
/s/accomodate/accumulate/

Reply:
Yeah, I wish there was some kind of "squash" feature as most Ruby app servers tend to restart workers from time to time.